### PR TITLE
chore: update to the latest Looker components

### PR DESCRIPTION
**Pull request by jkaster**
Also updated related packages

### CODE REVIEW

File: packages/api-explorer/src/components/Markdown/common.tsx [hunk 1]
```diff
@@ -43,14 +43,11 @@ MDCode.defaultProps = {
   pr: 'xxsmall',
 }
 
-// Temp fix for release build failure
-export const MDCodeBlock = styled(CodeBlock)`
+export const MDCodeBlock = styled(CodeBlock).attrs(
+  ({ fontSize = 'small', mb = 'large' }) => ({ fontSize, mb })
```
Comment (noahamac) [inline packages/api-explorer/src/components/Markdown/common.tsx:47]: Is this a new recommended pattern?

Comment (lukelooker) [reply packages/api-explorer/src/components/Markdown/common.tsx:47]: Definitely within the @looker/components code-base it is :)

See: https://dev.to/layershifter/how-to-kill-tree-shaking-in-webpack-with-static-properties-p72

Comment (noahamac) [reply packages/api-explorer/src/components/Markdown/common.tsx:47]: Thanks for the link!

**Review by noahamac — APPROVED** [review]

File: packages/api-explorer/src/components/Markdown/common.tsx (hunk 1 above)
Comment (jkaster) [reply packages/api-explorer/src/components/Markdown/common.tsx:47]: I guess, because @lukelooker recommended it!

Comment (lukelooker) [reply packages/api-explorer/src/components/Markdown/common.tsx:47]: I'm pretty sure that answer is almost as good as "I saw it on the internet so it _must_ be true" 😄

**Review by bryans99 — APPROVED** [review]
LGTM as well

**Pull request merged**